components/Topic.js

```diff
@@ -1,7 +1,21 @@
 import React from "react";
-
-const Topic = () => {
-	return <div></div>;
+const Topic = ({ topic, backgroundColor, textColor }) => {
+	return (
+		<div>
+			<div
+				className='flex justify-center items-center w-96 h-36 rounded-3xl border-2 border-black mb-8 shadow-black'
+				style={{
+					backgroundColor: backgroundColor,
```

Comment:
use Tailwind here, you can do `bg-[backgroundColor]` the same can be done for text `text-[textColor]`